pkg/realmd/README.md

```diff
@@ -23,3 +25,65 @@ Make sure this works:
 
 And now you're ready to use the feature. There's an account called
 "admin" with the password "foobarfoo".
+
+To test your DNS, the following should succeed without any error messages
+on both your client machines, and your server with cockpit:
```

Comment:
both client machines in addition to the server?
Otherwise I think this would be better:
`on your server with cockpit and the client machines:`